src/main/resources/changesets/changelog-consent-99.0.xml

```diff
@@ -1,7 +1,7 @@
 <databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
                    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
          http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.1.xsd">
-    <changeSet id="99.0" author="connorlbari=k">
+    <changeSet id="99.0" author="connorlbark">
```

Comment:
Changes to existing change sets that have been run in an environment will throw checksum errors. Liquibase hashes the file and stores the result in a table. If they don't match, LB chokes. To get around this, you have to mark changed files with:
```
<validCheckSum>ANY</validCheckSum>
```
There are quite a few that I modified when migrating from mysql to postgres. See https://www.liquibase.com/blog/what-affects-changeset-checksums for more info.